src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -66,16 +68,23 @@ export const DepositTx: FC<DepositTxProps> = ({
   const expectedTxOutcome = useAppSelector(VaultsSelectors.selectExpectedTxOutcome);
   const expectedTxOutcomeStatus = useAppSelector(VaultsSelectors.selectExpectedTxOutcomeStatus);
   const actionsStatus = useAppSelector(VaultsSelectors.selectSelectedVaultActionsStatusMap);
+  const selectedTokenMap = useAppSelector(TokensSelectors.selectTokensMap);
 
   const sellTokensOptions = selectedVault
     ? [selectedVault.token, ...userTokens.filter(({ address }) => address !== selectedVault.token.address)]
     : userTokens;
   const sellTokensOptionsMap = keyBy(sellTokensOptions, 'address');
-  const selectedSellToken = sellTokensOptionsMap[selectedSellTokenAddress ?? ''];
+  let selectedSellToken: TokenView | undefined = sellTokensOptionsMap[selectedSellTokenAddress ?? ''];
```

Comment:
I think we should probably try and refactor this `sellTokensOptionsMap[selectedSellTokenAddress ?? '']` so that if there isn't a `selectedSellTokenAddress` we don't get here or handle it more gracefully.

`sellTokensOptionsMap['']` should not be an option in the first place, what do you think @huggingbot?

Reply:
Thanks @huggingbot! Looking better and safer 🙇